retrieval_service/models/example-config-spanner.yml

```diff
@@ -0,0 +1,8 @@
+host: 0.0.0.0
+datastore:
+  # Example for Spanner
+  kind: "spanner"
+  project: "my-project"
+  instance: "my-instance"
+  database: "my_database"
+  service_account_key_file: "path/service/account/credentials.json"
```

Comment:
Can we move this into the existing config example? We can include examples of multiple databases and just comment them out until used

Reply:
Thanks for the review, currently two example config are present, do we want to merge them into one? 
- retrieval_service/example-config-cloudsql.yml
- retrieval_service/example-config.yml

Reply:
Probably -- I think including the same content just commented out makes the most sense imo